plugins/basic/python/tk_maya_basic/plugin_logic.py

```diff
@@ -0,0 +1,207 @@
+# Copyright (c) 2016 Shotgun Software Inc.
```

Comment:
Actually, I was even tempted to split `plugin_logic.py` into `plugin_logic.py` and `plugin_ui.py`. Do not forget that the plug-in is still evolving. There will be more than just bootstrapping it.

I am trying to keep the various plug-in parts separate logically in modules (high-level logic, low-level bootstrap, etc.) to avoid a _one-big-file_ plug-in, and because some of these modules may end up in a common plug-in core.

I could possibly rename:

- `plugin_engine.bootstrap()` to `plugin_engine.bootstrap_engine()`
- `plugin_engine.shutdown()` to `plugin_engine.shutdown_engine()`

but these new names sound somewhat redundant.